src/main/webapp/script.js

```diff
@@ -1,20 +1,16 @@
 function determineLogin() {
-  fetch('/login').then(response => response.json()).then((isLoggedIn) => {
-    console.log("Displaying login info")
+  fetch('/get-login-info').then(response => response.json()).then((isLoggedIn) => {
     const element = document.getElementById('form');
     const textElement = document.createElement('span');
     const form = document.createElement('form');
     form.method='GET';
     form.action='/login';
     const button = document.createElement('button');
     if (!isLoggedIn) {
-      textElement.innerText = "Please log in to view this page";
       button.innerHTML = "Login";
-      console.log("not logged in");
     } else {
-      textElement.innerText = "You're logged in!";
+      displayEmail();
```

Comment:
Instead of having `displayEmail()`, user email could be fetched from /get-login-info

Reply:
I agree, I will make this in an upcoming pull request by trying to merge the servlets